src/components/SamlProviderConfiguration/SamlProviderConfigForm.jsx

```diff
@@ -76,10 +124,37 @@ class SamlProviderConfigForm extends React.Component {
     }
   }
 
+  renderField = data => {
+    const { invalidFields } = this.state;
+    const { config } = this.props;
+    return (
+      <Form.Group
+        controlId={data.key}
+        isInvalid={invalidFields[data.key] || invalidFields[data.invalidAdditionalCondition]}
+      >
+        <Form.Label htmlFor={data.key}>{data.label}{data.showRequired && <span className="required">*</span>}</Form.Label>
+        <Form.Control
+          type={data.type || 'text'}
+          id={data.key}
+          name={data.key}
+          // eslint-disable-next-line no-nested-ternary
+          defaultValue={config ? config[data.key] : data.type === 'number' ? 1 : ''}
+          data-hj-suppress
+        />
+        <Form.Text>{data.helpText}</Form.Text>
```

Comment:
Perhaps to make this function a bit more flexible, should we conditionally render `Form.Text` only if `data.helpText` is provided to make it act more optional?